.travis.yml

```
@@ -7,16 +7,16 @@ script:
 - pip install -U planemo flake8 xmldiff
 - pip install -r requirements.txt
 - python setup.py install
-- flake8 argparse argparse2tool click
-- PYTHONPATH=$(argparse2tool_check_path -q) python examples/example.py --generate_galaxy_xml > tmp.xml
+- flake8 argparse2tool
+- PYTHONPATH=$(argparse2tool_check_path) python examples/example.py --generate_galaxy_xml > tmp.xml
```

Maybe we should just do:

```suggestion
- PYTHONPATH=$(argparse2tool) python examples/example.py --generate_galaxy_xml > tmp.xml
```

everywhere since checking the path doesn't make so much sense, and we only have one entrypoint in the setup.

---

I always thought `PYTHONPATH` needs to be a (absolute) path. We could rename `argparse2tool_check_path` to something more appropriate, e.g. `argparse2tool_get_path`. Using a script to obtain the installation path would cover as well cases where the package is installed with `--prefix`, I guess.

`$(argparse2tool)` apparently just returns nothing.

---

Oh I think I miscommunicated. I meant re-naming the console entry point in setup.py from `argparse2tool_check_path` to just `argparse2tool`, something more appropriate. I doubt we will ever have other entry points, so why not use the name of our package.